.gitlab/kitchen_testing/centos.yml

```diff
@@ -49,14 +49,14 @@
 
 .kitchen_scenario_centos_all_non_fips_a6:
   extends:
-    - .kitchen_os_centos_all_non_fips
     - .kitchen_agent_a6
+    - .kitchen_os_centos_all_non_fips
```

Comment:
I think we should add a comment somewhere (`.kitchen_agent_a6` definition?) about the need for the order

Reply:
Added a comment in the kitchen docs.